util/regexp-assemble/regexp-941130.data

```diff
@@ -1,3 +1,19 @@
+##! This is a data file used to generate a regular expression for a CRS rule. 
+##! The generation of the regular expression happens with the help of 
+##! util/regexp-assemble/regexp-assemble.pl.
+##! The ID of the rule in question is part of the file name of this data file. 
+##! Read more about the format of this data file and the use of the assembly 
+##! script in util/regexp-assemble/README.md.
+##!
+##! Lines starting with `##!` are comments and will be skipped,
+##! empty lines will be ignored completely.
+##! 
+##! Three special comments are at your disposal to influence the assembled expression:
+##! - `##!+`: the flag comment
+##! - `##!^`: the prefix comment
+##! - `##!$`: the suffix comment
+##! Please refer to util/regexp-assemble/README.md for a full explanation.
+
```

Comment:
The generated regex and the old rule differ. I think this is not a problem. It's the same, but we should replace the generated regex in the rule file.

```
$ diff 941130.old 941130.new
1,2c1
< (?i)[\s\S](?:!ENTITY\s+(?:\S+|%\s+\S+)\s+(?:PUBLIC|SYSTEM)|x(?:link:href|html|mlns)|data:text/html|pattern\b.*?=|formaction|\@import|;base64)\b
< 
---
> (?i)[\s\S](?:!ENTITY\s+(?:\S+|%\s+\S+)\s+(?:PUBLIC|SYSTEM)|x(?:link:href|html|mlns)|data:text\/html|pattern\b.*?=|formaction|;base64|@import)\b
```

Reply:
Hm. More missing escapes....

Reply:
After updating the script this regex is now identical except for an additional escape. I think that's ok.

Reply:
I checked the output of the original script again and the additional backslash before `/html` is present there too. So the regex in the rule must have been modified by hand. Both scripts yield the same expression.